src/components/Input/BaseInputLayout/styles.ts

```diff
@@ -19,68 +19,61 @@ const contentMargin = {
   medium: 12,
 } as const;
 
-export default createUseStyles(
-  (
-    theme,
-    hasLabel: boolean,
-    inputExists: boolean,
-    size: Required<IBaseInputLayoutProps>['size']
-  ) => ({
-    baseInput: {
-      height: height[size],
-      borderWidth: inputConstants.BORDER_WIDTH,
-      borderColor: theme.grey.main,
-      borderRadius: 4,
-      paddingHorizontal: paddingHorizontal[size],
-      flexDirection: 'row',
-      justifyContent: 'space-between',
-      alignItems: 'center',
-    },
-    baseInputFocused: {
-      borderColor: theme.primary.main,
-    },
-    baseInputDisabled: {
-      borderColor: theme.grey.light,
-    },
-    childrenContainer: {
-      flex: 1,
-      marginBottom: hasLabel ? 6 : 0,
-      alignItems: 'flex-start',
-      justifyContent: hasLabel ? 'flex-end' : 'center',
-    },
-    label: {
-      color: theme.text.primary,
-      fontFamily: theme.fontFamily.regular,
-      fontWeight: '400',
-      fontSize: 16,
-      lineHeight: 24,
-      position: 'absolute',
-      left: 0,
-      top: 16,
-      textAlignVertical: 'center',
-    },
-    labelSmall: {
-      color: theme.text.hint,
-    },
-    error: {
-      borderColor: theme.error.main,
-    },
-    errorFocusedOutline: {
-      backgroundColor: theme.error.light,
-    },
-    labelDisabled: {
-      color: theme.text.disabled,
-    },
-    mainContent: {
-      alignSelf: 'stretch',
-      flex: 1,
-    },
+export default createUseStyles((theme, hasLabel: boolean, inputExists: boolean) => ({
```

Comment:
These changes break the select with only icons and the phone select, which is based on this type of select.

Reply:
yeah. I am sizing it based on the label. I'm going to add a size prop with small and medium options to overcome that

Reply:
Resolved